levels/intro/geo.c

```diff
@@ -37,9 +37,9 @@ const GeoLayout intro_geo_0002D0[] = {
          GEO_CLOSE_NODE(),
       GEO_CLOSE_NODE(),
       GEO_ZBUFFER(0),
-      GEO_OPEN_NODE(),
-         GEO_ASM(0, geo_fade_transition),
-      GEO_CLOSE_NODE(),
+ //     GEO_OPEN_NODE(),
```

Comment:
is this deliberate? is it psp specific?

Reply:
Any actual Mario engines are accidental and shouldn't have been committed. 
Thank you for understanding. Only changes to Makefile and code under PC.